Configurations/50-win-onecore.conf

```diff
@@ -82,7 +82,35 @@ my %targets = (
         ex_libs         => "onecore.lib",
         multilib        => "-arm64",
     },
-
+    "VC-WIN64-CLANGASM-ARM" => {
```

Comment:
I would rather have your definitions in a separate file, say `Configurations/50-win-clang-cl.conf`